src/main/resources/templates/schema-management.html

```diff
@@ -53,14 +53,18 @@ <h4>Schema Record Form</h4>
         options = /*[[${request}]]*/ {};
         /*]]>*/
 
+        var table = {
```

Comment:
Is it possible to put all static variables in a separate (html?) file and then include it in all other html pages?

Reply:
> Is it possible to put all static variables in a separate (html?) file and then include it in all other html pages?

Which static variables do you mean here ? variable table?

Reply:
No, sorry for wrong anchor. 
All static variables like prefixVersion, basePathSchema, ...